crates/ruff_linter/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -81,13 +82,27 @@ impl Violation for PytestParametrizeNamesWrongType {
 
     #[derive_message_formats]
     fn message(&self) -> String {
-        let PytestParametrizeNamesWrongType { expected } = self;
-        format!("Wrong name(s) type in `@pytest.mark.parametrize`, expected `{expected}`")
+        let PytestParametrizeNamesWrongType {
+            single_argument,
+            expected,
+        } = self;
+        if *single_argument {
+            return format!(
+                "Wrong type passed to first argument of `@pytest.mark.parametrize`; expected `str`"
+            );
+        }
+        format!("Wrong type passed to first argument of `@pytest.mark.parametrize`; expected `{expected}`")
```

Comment:
Hmm, I don't think we should put backticks around `{expected}` here, since `string of comma-separated values` is a prose description rather than code. I'd remove the backticks here...

```suggestion
        format!("Wrong type passed to first argument of `@pytest.mark.parametrize`; expected {expected}")
```